insights/parsers/lvm.py

```diff
@@ -92,6 +93,8 @@ class Lvm(Parser):
     """Base class for parsing LVM data in key=value format."""
 
     def parse_content(self, content):
+        if "Unrecognised field:" in content[-1]:
```

Comment:
@chenlizhong I'm wondering if there should be a more generalized solution to this problem?  This message string is very specific and to some extent localized.

Reply:
@bfahr 
Look at the source code of lvm.  "Unrecognised field:" is hardcode.  I also tested vgs, pvs, lvs they all give "Unrecognised field:" when the -o parameter following unsupport field.

~~~
if (!_field_match(rh, ws, (size_t) (we - ws), report_type_only)) {
    _display_fields(rh, 1, 0);
    log_warn(" ");
    log_error("Unrecognised field: %.*s", (int) (we - ws), ws);
~~~

https://github.com/lvmteam/lvm2/search?utf8=%E2%9C%93&q=Unrecognised+field&type=

Reply:
Thank you @chenlizhong, that seems more reasonable then.

Reply:
@chenlizhong - how about changing to ParseException?